.github/workflows/ant.yml

```diff
@@ -0,0 +1,22 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Install and run ipfs
+      run: ./install-run-ipfs.sh
+    - name: Build with Ant
+      run: ant -noinput -buildfile build.xml dist
+    - name: Run tests
+      timeout-minutes: 10
+      run: ant -noinput -buildfile build.xml test
```

Comment:
Looks like this task runs [only one](https://github.com/ipfs-shipyard/java-ipfs-http-client/runs/921407072?check_suite_focus=true#step:6:20) of the test sites. What about the [other](https://github.com/ipfs-shipyard/java-ipfs-http-client/tree/fix/ipfs-v0.5.0/src/test/java/io/ipfs/api) ones?

Reply:
It runs runs the main test suite, which is what it's always done, but I've expanded it nonetheless.